ipaserver/install/server/upgrade.py

```diff
@@ -1955,6 +1965,7 @@ def upgrade_configuration():
     cleanup_kdc(fstore)
     cleanup_adtrust(fstore)
     cleanup_dogtag()
+    cleanup_kdcinfo()
```

Comment:
Please move call to `cleanup_kdcinfo()` into `sssd_update()` because they need to be coordinated -- do it as a last step there.

Reply:
@abbra thanks for review. I have replaced `cleanup_kdcinfo` calling to `sssd_update` before sssd restart, actually it isn't a last step inside the function, but i guess it's better to do before service restart. Check updated commit please.